src/easyLess.ts

```diff
@@ -19,7 +22,7 @@ export function activate(context: vscode.ExtensionContext) {
 
       if (document.fileName.endsWith(LESS_EXT)) {
         document.save();
-        new CompileLessCommand(document, lessDiagnosticCollection).execute();
+        new CompileLessCommand(document, lessDiagnosticCollection).setPreprocessor(preprocessor).execute();
```

Comment:
Following on from above, this should become `setPreprocessors()` (plural), and also on lines 37/44.